MozziGuts.cpp

```diff
@@ -89,6 +92,12 @@ CircularBuffer <unsigned int> output_buffer2; // fixed size 256
 #endif
 
 #if IS_AVR() // not storing backups, just turning timer on and off for pause for teensy 3.*, other ARMs
+#ifdef __DAC_MCP__   // in case an external MCP4922 dac is used.
```

Comment:
Just an idea: If you move this function to the AudioConfig(DAC).h-file, it will be easier to support additional DAC without even touching MozziGuts.cpp, in the future.

What about also supporting stereo output, though (`#if (STEREO_HACK == true)`)? From the data sheet, the MCP9422 is dual channel, so that would be an obvious use case, no? Perhaps define one output function for mono, and a separate for stereo.